core/kazoo_services/src/kz_services.erl

```diff
@@ -592,7 +592,11 @@ select_bookkeeper(BillingId) ->
     ResellerId = get_reseller_id(BillingId),
     {'ok', MasterAccountId} = kapps_util:get_master_account_id(),
     case ResellerId =/= MasterAccountId of
-        'true' -> 'kz_bookkeeper_local';
+        'true' ->
+            case ?KZ_RESELLER_BOOKKEEPER_LOOKUP of
+                'true' -> ?KZ_ACCOUNT_BOOKKEEPER(BillingId);
```

Comment:
@onnet Howdy, this is a very nice addition, however I would expect KZ_ACCOUNT_BOOKKEEPER to look up the from the ResellerId rather than the BillingId. Using the BillingId would allow each subaccount to have a different bookkeeper, but it would be our intention that the bookkeeper be configured at the reseller level.

IE: If an account is flagged as a reseller then they can set the bookkeeper they would like to use on their account, and it would implicitly apply to all sub-accounts. As it stands it would have to be configured on each of the resellers subaccounts.

Let us know what you think, thanks!